tools/collect_logs/collect_all_logs

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env ruby
+require File.expand_path("../../config/environment", __dir__)
+
+user = 'root' # TODO accept user provided argument for username
+
+MiqServer.active_miq_servers.each do |miq_server|
```

Comment:
Wondering if there should be some kind of sort here for some determinism.  Not sure _what_ we would sort on, though.

Reply:
Does determinism matter as long as we get all of them?  They're going into a directory which will be sorted by the filesystem based on the name not sure it matters what order they're collected in

Reply:
It's more of a UX thing for the person watching the script.  It helps the person running it know they are roughly on x out of y, or if it's alphabetical they know they are reaching the end.

Reply:
If that's the case maybe a progress counter like "...Complete - X out of Y" would be better then

Reply:
oh yeah that might be better.